app/main.py

```diff
@@ -1,3 +1,54 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance(km={self.km})"
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if isinstance(other, int) or isinstance(other, float):
```

Comment:
```suggestion
        if isinstance(other, (int,float)):
```
This condition could be simplified